builder/main.go

```diff
@@ -23,7 +23,7 @@ func main() {
 
 	runner := buildpackrunner.New(&config)
 
-	err := runner.Run()
+	_, err := runner.Run()
```

Comment:
Personally, I don't have a huge problem with treating the `buildpackrunner` as a library for (our own) external use in this context.